tools/cli/commands-cordova.js

```diff
@@ -131,6 +133,7 @@ main.registerCommand({
   requiresApp: true,
   catalogRefresh: new catalog.Refresh.Never()
 }, function (options) {
+  ensureDevBundleDependencies();
```

Comment:
Oh I see now why you didn't just pass `doAddPlatform` to `main.registerCommand`. Sorry for the noise.